pkg/api/team_members.go

```diff
@@ -29,7 +30,8 @@ func (hs *HTTPServer) GetTeamMembers(c *models.ReqContext) response.Response {
 
 	filteredMembers := make([]*models.TeamMemberDTO, 0, len(query.Result))
 	for _, member := range query.Result {
-		if dtos.IsHiddenUser(member.Login, c.SignedInUser, hs.Cfg) {
+		// TODO: when FGAC is enabled, use SQL filtering to filter out for users that the caller has permissions to see
+		if !hs.Cfg.FeatureToggles["accesscontrol"] && dtos.IsHiddenUser(member.Login, c.SignedInUser, hs.Cfg) {
```

Comment:
I think hidden users should be hidden even if you have the permission to view it.